bcda/api.go

```diff
@@ -182,14 +190,16 @@ func jobStatus(w http.ResponseWriter, r *http.Request) {
 	i, err := strconv.Atoi(jobID)
 	if err != nil {
 		log.Print(err)
-		http.Error(w, http.StatusText(400), 400)
+		oo := responseutils.CreateOpOutcome(responseutils.Error, responseutils.Exception, "", responseutils.Processing)
+		responseutils.WriteError(oo, w, http.StatusBadRequest)
 		return
 	}
 	var job models.Job
 	err = db.First(&job, i).Error
 	if err != nil {
 		log.Print(err)
-		http.Error(w, http.StatusText(404), 404)
+		oo := responseutils.CreateOpOutcome(responseutils.Error, responseutils.Exception, "", responseutils.DbErr)
+		responseutils.WriteError(oo, w, http.StatusInternalServerError)
```

Comment:
Interesting. So I've always thought about 404 being like a missing web page due to a bad url link or something. But maybe it can also represent a missing job in our database?? Not opposed to changing this